hello-world-master/source/components/scene/descriptionscreen.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version = "1.0" encoding = "utf-8" ?>
+ 
+<component name = "DescriptionScreen" extends = "Group">
+  <script type="text/brightscript" uri="descriptionscreen.brs"/>
+   <interface>
+    <field id = "content" type = "node" onChange = "showcontent"/>
+  </interface>
+  <children >
+    <Poster 
+      id = "itemPoster" 
+      width = "1920" 
+      height = "1080" />
+    <Label 
+    id="description"
+    height="500"
+    width="1720"
+    font="font:MediumBoldSystemFont"
+    text = ""
+    horizAlign = "left"
+    vertAlign = "center"
+    translation="[100,500]"
+    wrap = "true"/>
```

Comment:
Indentation. If you look closely, the colors are different if you let spaces between `=` signs.
You can remove the extra spaces here